test/e2e/constants/constants.go

```diff
@@ -26,7 +26,7 @@ var (
 	Memory            = constants.GetEnvWithDefault("INFINISPAN_MEMORY", "512Mi")
 	Namespace         = strings.ToLower(constants.GetEnvWithDefault("TESTING_NAMESPACE", "namespace-for-testing"))
 	RunLocalOperator  = strings.ToUpper(constants.GetEnvWithDefault("RUN_LOCAL_OPERATOR", "true"))
-	RunSaOperator     = strings.ToUpper(constants.GetEnvWithDefault("RUN_SA_OPERATOR", "true"))
+	RunSaOperator     = strings.ToUpper(constants.GetEnvWithDefault("RUN_SA_OPERATOR", "false"))
```

Comment:
@rigazilla please pay attention, that problem is that RunSaOperator was always set to true, instead of false

Reply:
uh yes @dmvolod , wrong default. my bad